src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -69,7 +69,7 @@ void Tailsitter::update_vtol_state()
 	 * For the backtransition the pitch is controlled in MC mode again and switches to full MC control reaching the sufficient pitch angle.
 	*/
 
-	float pitch = Eulerf(Quatf(_v_att->q)).theta();
+	const float pitch = Eulerf(Quatf(_v_att->q)).theta();
```

Comment:
This could be moved into the `case vtol_mode::TRANSITION_BACK `case below, no need to keep it outside of the switch.